pkg/wrapper/nmt_wrapper_test.go

```diff
@@ -149,3 +166,34 @@ func generateRandNamespacedRawData(count int) (result [][]byte) {
 func sortByteArrays(src [][]byte) {
 	sort.Slice(src, func(i, j int) bool { return bytes.Compare(src[i], src[j]) < 0 })
 }
+
+// TestErasuredNamespacedMerkleTree_ProveRange checks that the proof returned by the ProveRange for all the shares within the erasured data is non-empty.
```

Comment:
[extremely optional]

```suggestion
// TestErasuredNamespacedMerkleTree_ProveRange verifies the proof returned by
// ProveRange for all shares ranges of size 1.
```